src/main/scripts/org/reaktivity/specification/nukleus/tcp/streams/server.sent.data.and.close/client/nukleus.rpt

```diff
@@ -0,0 +1,85 @@
+#
```

Comment:
This scenario should be named `server.sent.data.then.end` instead of `server.sent.data.and.close`.